change-beta/@azure-communication-react-df2fc150-ffbd-4e1a-bb90-ad1ea5304bf2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
If this is for Captions GA is this pre-release?

Reply:
It's feature related changes, I think it should be prerelease?